### DISCUSSION TITLE
Wayand Support

### DISCUSSION THREAD

**Discussion by memmel2** [o3de/o3de#15647] [General]
I'm interested in porting o3de too wayland. Basically don't know who I should talk with.

**Comment by AMZN-daimini** [comment]
Hi there! I am not familiar with Wayland, but I would recommend you to join the O3DE Discord (https://discord.com/invite/o3de) and ask in the sig-build and/or sig-network rooms, as that is where you're most likely to find someone that can help :)

**Comment by memmel2** [comment]
Sorry for the slow reply 

https://wayland.freedesktop.org/

I think you misunderstood the question.
